vic/vic_run/src/func_canopy_energy_bal.c

```diff
@@ -217,7 +217,7 @@ func_canopy_energy_bal(double  Tfoliage,
         /* Calculate the latent heat flux */
 
         Ls = calc_latent_heat_of_sublimation(Tfoliage);
-        *LatentHeatSub = Ls * *VaporMassFlux * CONST_RHOFW;
+        *LatentHeatSub = Ls **VaporMassFlux * CONST_RHOFW;
```

Comment:
OK - I am not going to argue with uncrustify (or I guess I am), This is harder to read then before. replace `**VaporMassFux` with `* (*VaporMassFlux)` to separate the multiplier (first `*`) from the pointer dereferencing (second `*`)